docs/source/user_guide/installation.md

```diff
@@ -83,37 +117,65 @@ This is equivalent to running `napari -w movement` and should open the `napari`
 window with the `movement` widget docked on the right-hand side.
 
 ## Update the package
-To update `movement` to the latest version, you need to use the same package manager you used to install it (either `conda` or `pip`). If you are not sure which one you used, you can run from your active environment:
+
+Always update using the same package manager you used for installation.
+
+If you are not sure which one you used, you can run `conda list movement` from your active `conda` environment
+and check whether the output shows `conda-forge` or `pypi` as the channel.
+
+To update `movement`, run the appropriate command below:
+
+::::{tab-set}
+:::{tab-item} conda
 ```sh
-conda list movement
+conda update -c conda-forge movement -y
 ```
-If the output shows `conda-forge` as the channel, you used `conda` to install it. If it shows `pypi`, you used `pip`.
+:::
 
-Once this is clear, you can update `movement` by running the relevant command from below:
-::::{tab-set}
-:::{tab-item} Conda
+:::{tab-item} pip
 ```sh
-conda update movement -y
+pip install -U movement
 ```
 :::
 
-:::{tab-item} Pip
+:::{tab-item} uv
 ```sh
-pip install movement -U
+uv pip install -U movement
 ```
 :::
 ::::
 
 
-If the above fails, try installing it in a fresh new environment instead. This should prevent potential compatibility issues caused by changes in dependency versions. You can first uninstall the existing environment named `movement-env`:
+If the above fails, try installing `movement` in a fresh new environment to avoid dependency conflicts.
+
+First remove the existing environment:
+
+::::{tab-set}
+:::{tab-item} conda
 ```sh
 conda env remove -n movement-env
 ```
-Once the environment has been removed, you can create a new one following the [installation instructions](#install-the-package) above.
 
-:::{tip}
-If you are unsure about the environment name, you can get a list of the environments on your system with:
+This command assumes your environment is named `movement-env`.
+If you are unsure about the name, you can get a list of the environments
+on your system with `conda env list`.
+:::
+
+:::{tab-item} uv
+Simply delete the `.venv` folder in your project directory.
+
+```bash
+rm -rf .venv       # On macOS and Linux
+```
+```powershell
+rmdir /s /q .venv  # On Windows PowerShell
+```
```

Comment:
````suggestion
```powershell
rm -rf .venv       # On macOS and Linux
rmdir /s /q .venv  # On Windows PowerShell
```
````